mk_verificator/tests/system/test_ntp_sync.py

```diff
@@ -1,89 +1,52 @@
-import salt.client as client
-import texttable as tt
+import pytest
+import json
+from mk_verificator import utils
 
-data_to_draw = {}
-node_times_list = []
-h = 0
-m = 0
-s = 0
-h_gauge = 0
-m_gauge = 0
-s_gauge = 5
 
+@pytest.mark.parametrize(
+    ("group"),
+    utils.get_groups(utils.get_configuration(__file__))
+)
+def test_ntp_sync(group, local_salt_client):
 
-def count_avg_time(node_times_list):
-    global h, m, s
-    divisor = -1
-    for time_list in node_times_list:
-        h += int(time_list[0])
-        m += int(time_list[1])
-        s += int(time_list[2])
-        divisor += 1
-    h = h / divisor
-    m = m / divisor
-    s = s / divisor
-
-
-def draw_results_table(data_to_draw):
-    print "Trying to draw a table with the results..."
-    global h, m, s, h_gauge, m_gauge, s_gauge
-
-    tab = tt.Texttable()
-    tab.set_chars(['-', '|', '+', '-'])
-    tab.set_cols_align(["c", "c", "c", "c"])
-    tab.set_cols_valign(["c", "c", "c", "c"])
-    tab.add_row(["Node", "Time", "AVG time", "Result"])
+    data = {}
+    node_times_list = []
+    hour = 0
+    minute = 0
+    second = 0
+    hour_gauge = 0
+    minute_gauge = 0
+    second_gauge = 5
+    divisor = 0
 
-    fail_tab = tt.Texttable()
-    fail_tab.set_chars(['-', '|', '+', '-'])
-    fail_tab.set_cols_align(["c", "c", "c", "c"])
-    fail_tab.set_cols_valign(["c", "c", "c", "c"])
-    fail_tab.add_row(["Node", "Time", "AVG time", "Result"])
-
-    for node in data_to_draw:
-        ntime = data_to_draw.get(node)
-
-        n_time = "{}h {}m {}s".format(ntime[0],
-                                      ntime[1],
-                                      ntime[2])
-        tgauge = "{}h {}m {}s".format(h, m, s)
-
-        if (int(ntime[0]) - h) != h_gauge:
-            fail_tab.add_row([node, n_time, tgauge, "FAILED"])
-        elif (int(ntime[1]) - m) != m_gauge:
-            fail_tab.add_row([node, n_time, tgauge, "FAILED"])
-        elif (int(ntime[2]) - s) > s_gauge:
-            # TODO: add correct verification for seconds difference
-            fail_tab.add_row([node, n_time, tgauge, "FAILED"])
-        else:
-            tab.add_row([node, n_time, tgauge, "+"])
+    fail = {}
 
-    print "Table with results:"
-    print tab.draw()
+    nodes_info = local_salt_client.cmd(group, 'cmd.run', ['date +"%H %M %S"'])
```

Comment:
You can use 'date +"%s" ' maybe it will be more comfortable than 'date +"%H %M %S"'